src/main/java/io/jenkins/plugins/pipelinegraphview/utils/FlowNodeWrapper.java

```diff
@@ -129,12 +130,12 @@ public WorkflowRun getRun() {
     }
 
     public @NonNull String getDisplayName() {
-        // Make 'PARALLEL_BLOCK' types have the display name as SytheticNodes used to.
+        // Make 'PARALLEL_BLOCK' types have the display name as SyntheticNodes used to.
         if (type == NodeType.PARALLEL_BLOCK) {
-            return "Parallel";
+            return Messages.FlowNodeWrapper_parallel();
         }
         if (type == NodeType.PIPELINE_START) {
-            return "Unhandled Exception";
+            return "%s".formatted(Messages.FlowNodeWrapper_noStage());
         }
         return displayName;
```

Comment:
github is getting confused with me making a suggestion for this whole method but thoughts on

```java
        return switch (type) {
            // Make 'PARALLEL_BLOCK' types have the display name as SyntheticNodes used to.
            case PARALLEL_BLOCK -> Messages.FlowNodeWrapper_parallel();
            case PIPELINE_START -> Messages.FlowNodeWrapper_noStage();
            default -> displayName;
        };
```

Reply:
much simpler 👍